R/info.R

```diff
@@ -0,0 +1,21 @@
+#' Get information about PROOF server
+#'
+#' @export
+#' @references <https://github.com/FredHutch/proof-api?tab=readme-ov-file#get-info>
+#' @return A list with fields:
+#' - `logged_in_user` (character): NULL unless authentication header is sent
+#' - `api_endpoint` (character): the URL for the PROOF API
```

Comment:
in the response I see when running this function the url has a `http` scheme rather than `https`. Is that on purpose?

Reply:
Thanks for letting me know. This is an issue with the API code. I have filed an issue about it (see below).

Reply:
Thanks

Reply:
@dtenenba were you going to merge https://github.com/FredHutch/proof-api/pull/63 at some point? And it seems like https://github.com/FredHutch/proof-api/issues/62 should still be open given that https://github.com/FredHutch/proof-api/pull/63 is open

Reply:
@sckott -- Just merged https://github.com/FredHutch/proof-api/pull/63 and removed `api_endpoint` from the dev branch of proof-api if you want to implement this.

Reply:
Thanks. @dtenenba when will the changes on proof-api dev merge into main?